app/views/merchant/items/index.html.erb

```diff
@@ -10,10 +10,11 @@
       <%= image_tag(item.image) %>
       <li>Active?: <%= item.status %></li>
       <li>Inventory: <%= item.inventory %></li>
+      <%= link_to "Edit Item", "/merchant/items/#{item.id}/edit", method: :get %>
       <% if item.status == true %>
```

Comment:
We should add a `enum` for status as well.. `enum status: { active: true, inactive: false }`